Makefile

```diff
@@ -32,16 +32,16 @@ EXE_EXT 	= .exe
 endif
 
 # Interrogate the git repo and set some variables
-REPO_ROOT 		   		 = $(shell git rev-parse --show-toplevel)
-REVISION 		   		?= $(shell git rev-parse --short HEAD)
-ACN_VERSION  	   		?= $(shell git describe --exclude "azure-ipam*" --exclude "dropgz*" --exclude "zapai*" --tags --always)
-AZURE_IPAM_VERSION 		?= $(notdir $(shell git describe --match "azure-ipam*" --tags --always))
-CNI_VERSION        		?= $(ACN_VERSION)
-CNI_DROPGZ_VERSION 		?= $(notdir $(shell git describe --match "dropgz*" --tags --always))
-CNI_DROPGZ_TEST_VERSION ?= $(notdir $(shell git describe --match "dropgz-test*" --tags --always))
-CNS_VERSION  	   		?= $(ACN_VERSION)
-NPM_VERSION        		?= $(ACN_VERSION)
-ZAPAI_VERSION  	   		?= $(notdir $(shell git describe --match "zapai*" --tags --always))
+REPO_ROOT				 = $(shell git rev-parse --show-toplevel)
+REVISION				?= $(shell git rev-parse --short HEAD)
+ACN_VERSION				?= $(shell git describe --exclude "azure-ipam*" --exclude "dropgz*" --exclude "zapai*" --tags --always)
+AZURE_IPAM_VERSION		?= $(notdir $(shell git describe --match "azure-ipam*" --tags --always))
+CNI_VERSION				?= $(ACN_VERSION)
+CNI_DROPGZ_VERSION		?= $(notdir $(shell git describe --match "dropgz*" --tags --always))
+CNI_DROPGZ_TEST_VERSION	?= $(notdir $(shell git describe --match "dropgz-test*" --tags --always))
+CNS_VERSION				?= $(ACN_VERSION)
+NPM_VERSION				?= $(ACN_VERSION)
+ZAPAI_VERSION			?= $(notdir $(shell git describe --match "zapai*" --tags --always))
```

Comment:
nit: Is this an IDE change? If no additional commits happen lets fix this in the next PR

Reply:
To prevent adding additional comments for the same thing, the entire file has spacing issues.

Reply:
conventionally things in a Makefile are tab-aligned. where I have added to these block defs, I have fixed them to be tabbed.

Reply:
Tabbing issue came from a GH setting. Defaulted to 8 instead of 4. Thanks for standardizing to tab spacing.